app/models/endpoint.rb

```diff
@@ -3,6 +3,7 @@ class Endpoint < ApplicationRecord
 
   default_value_for :verify_ssl, OpenSSL::SSL::VERIFY_PEER
   validates :verify_ssl, :inclusion => {:in => [OpenSSL::SSL::VERIFY_NONE, OpenSSL::SSL::VERIFY_PEER]}
+  validates :port, :numericality => {:only_integer => true, :allow_nil => true, :greater_than => 0}
```

Comment:
Interesting...we've never had this problem before with any of our other numeric columns (including port which used to be a string), so I'm not sure why it's a problem now.  I guess the only other thing I'm questioning is whether or not this belongs as a validation at the model level, or if this is a UI concern text input concern.  @blomquisg Thoughts?